alembic/README

```diff
@@ -35,7 +35,7 @@ We use a few types not currently supported by SQLAlchemy, so you may
 need to modify the migration slightly.
 
 
-== Helper functions in socorro.lib.migrations
+== Helper functions in socorrolib.lib.migrations
```

Comment:
Probably a rogue search-and-replace. There is no `socorrolib.lib.migrations` or `socorro.lib.migrations` for that matter.